### ISSUE TITLE
____SIMPLE_HTML_DOM__VOKU__AMP____ Undefined constant

### ISSUE THREAD

**Issue by georgekaf** [voku/simple_html_dom#34]
#### What is this feature about (expected vs actual behaviour)?
Using the code below:
```
                        $dom = voku\helper\HtmlDomParser::file_get_html('5988.txt', true);
                        $element = $dom->findOne('h1.page-title');
                        print_r($element);
```

I get the `____SIMPLE_HTML_DOM__VOKU__AMP____` undefined constant (I guess?)
```
voku\helper\SimpleHtmlDom Object
(
    [node:protected] => DOMElement Object
        (
            [tagName] => h1
            [schemaTypeInfo] => 
            [nodeName] => h1
            [nodeValue] => 
      Proton Power ____SIMPLE_HTML_DOM__VOKU__AMP____amp; Light
    
            [nodeType] => 1
            [parentNode] => (object value omitted)
            [childNodes] => (object value omitted)
            [firstChild] => (object value omitted)
            [lastChild] => (object value omitted)
            [previousSibling] => (object value omitted)
            [nextSibling] => (object value omitted)
            [attributes] => (object value omitted)
            [ownerDocument] => (object value omitted)
            [namespaceURI] => 
            [prefix] => 
            [localName] => h1
            [baseURI] => 
            [textContent] => 
      Proton Power ____SIMPLE_HTML_DOM__VOKU__AMP____amp; Light
    
        )

)
```
#### How can I reproduce it?
Include the library through an optimised autoloader, generated by composer. 
Run the code with the file below. 

#### Does it take minutes, hours or days to fix?
I guess minutes if you know where to define the constant :) 

#### Any additional information?
I am including simple_html_dom through an optimised autoloader (generated with composer)
The file I am parsing is this:
[5988.txt](https://github.com/voku/simple_html_dom/files/3502542/5988.txt)

**Comment by voku** [comment]
please try e.g. ```print_r($element->html());``` or ```print_r($element->innerHtml());``` ... and take a look at the working examples in the tests: https://github.com/voku/simple_html_dom/blob/master/tests/HtmlDomParserTest.php

Happy Coding! :)

**Comment by georgekaf** [comment]
So I was using `$shopName = trim($element->textContent);` because I was expecting it would clear out everything related html tags and decode html entities as well.

So I replace it with something like that `$shopName = trim(htmlspecialchars_decode(strip_tags($element->innerhtml())));`

**Comment by voku** [comment]
fixed in version 4.7.x
